app/models/grant_submission/response.rb

```diff
@@ -31,9 +32,10 @@ class Response < ApplicationRecord
                                         foreign_key: 'grant_submission_multiple_choice_option_id',
                                         inverse_of: :responses,
                                         optional: true
-
     has_one_attached :document
 
+    has_date_optional_time(datetime_comp: :datetime_val, has_time_comp: :boolean_val)
```

Comment:
I realize this method was pulled in from Form Builder, but should this be here among the validations and/or should there be an `if question.type` check?